### ISSUE TITLE
Add PodGroup Admission

### ISSUE THREAD

**Issue by shinytang6** [volcano-sh/volcano#1372]
#### What would you like to be added:

If the podgroup created by the downstream job does not specify queue, the queue cannot recognize that podgroup when updating state.

**Comment by hudson741** [comment]
when the queue is not specified, the queue will be default?

**Comment by Thor-wl** [comment]
> when the queue is not specified, the queue will be default?

Yes

**Comment by shinytang6** [comment]
Yes, currently the "default" queue is set by default in the scheduler cache. If pg.spec.queue is empty, there will be no problem at scheduling level, but the "default" queue will not perceive the pg assigned to itself. So 
an enhanced way is through admission.

**Comment by shinytang6** [comment]
> when the queue is not specified, the queue will be default?

Yes, currently the "default" queue is set by default in the scheduler cache. If pg.spec.queue is empty, there will be no problem at scheduling level, but the "default" queue will not perceive the pg assigned to itself. So
an enhanced way is through admission.

**Comment by shinytang6** [comment]
added in https://github.com/volcano-sh/volcano/pull/1375
/close
